src/js/ipfsQueries.js

```diff
@@ -0,0 +1,20 @@
+import IpfsAPI from 'ipfs-api'
+
+class IpfsQueries {
+  constructor () {
+    this.ipfs = IpfsAPI()
+  }
+
+  getHash (hash) {
+    return this.ipfs.get(hash)
+  }
+
+  addNameUseHash (name, use) {
+    // TODO: Discuss 'salt' with team. Has pros & cons
+    const ipfsData = JSON.stringify({name: name, use: use, salt: Math.random()})
```

Comment:
@johnbayne I like this approach, @jsaur what do you think?
Either way agreed that we'll want the IPFS data to be represented in JSON

Reply:
+1

Reply:
Great, glad to hear. We can salt elsewhere down the line, but I wanted to introduce it here to ensure we didn't forget to do so. IPFS sends blocks of identical content to the same hash, so if a borrower put the same `use` for multiple loans, anyone who had access to the name/use of one loan could see all that borrower's loans that shared the same use.

On one hand, the salt protects against that. On the other, well - it seems slightly overkill. I'm happy either way. :-)